lib/src/logic/cruise.dart

```diff
@@ -8,7 +8,6 @@ import 'package:flutter/material.dart';
 import 'package:flutter/painting.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:meta/meta.dart';
-import 'package:pedantic/pedantic.dart' show unawaited;
```

Comment:
this exists because some error message somewhere told me to do it; did we fix the error message as well?

Reply:
The unawaited function is moving to package:meta, except maybe not. We should probably wait on landing this until that's settled. See https://github.com/flutter/flutter/pull/61052#issuecomment-655635438